src/resources/views/manual/tutorials/qgis-filtering-and-visualization.blade.php

```diff
@@ -0,0 +1,56 @@
+@extends('manual.base')
+
+@section('manual-title', 'Filtering data points in QGIS')
+
+@section('manual-content')
+  <div class="row">
+    <p class="lead">
+      QGIS include Filtering option through Attribute Table Interface and Query Builder Interface.
+    </p>
+    <p>
+      The response data of GeoJSON Api endpoints also includes Attributes of each point. These attributes are viewable in QGIS Attribute Table and <a href="https://docs.qgis.org/3.10/en/docs/user_manual/working_with_vector/attribute_table.html">numerous operations can be performed</a>. The Filtered points are represented with different color on the map.
+    </p>
+    <p>
+      One of the possible operation would be filtering data points by labels that occur in the images. This can be done through Attribute table.
+    </p>
+    <h3>Attribute Table</h3>
```

Comment:
As far as I understand, this part *selects* some features based on a filter query. This is useful to know but should be placed below the next section which is about the actual filtering. Also, the text should explicitly distinguish between "selection" and "filtering".